blocks/cards/cards.js

```diff
@@ -10,7 +10,29 @@ export default function decorate(block) {
     while (row.firstElementChild) li.append(row.firstElementChild);
     [...li.children].forEach((div) => {
       if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
+      else { div.className = 'cards-card-body'; }
+
+      /* cards simple */
+      if (block.classList.contains('simple')) {
+        const anchorEl = div.querySelector('a');
+        if (anchorEl) {
+          anchorEl.textContent = '';
+          div.append(anchorEl);
+          div.querySelector('.button-container').remove();
+        }
+        const cardBodyContent = document.createElement('div');
+        cardBodyContent.className = 'card-body-content';
+        if (anchorEl) {
+          anchorEl.append(cardBodyContent);
+        } else {
+          div.append(cardBodyContent);
+        }
+
+        [...div.querySelectorAll('.cards-card-body > :not(.card-body-content, a)')].forEach((el) => {
+          cardBodyContent.append(el);
+        });
+      }
+      /* END of cards simple */
```

Comment:
Rather than using a "start" + "end" of the block comment you might want to move it to a helper function for readability.

Reply:
Updated using processSimpleCard